kectl/app/cmd/util/k8sinstaller.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2019 The Kubeedge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import "fmt"
+
+//K8SInstTool embedes Common struct and contains the default K8S version and
+//a flag depicting if host is an edge or cloud node
+//It implements ToolsInstaller interface
+type K8SInstTool struct {
+	Common
+	IsEdgeNode     bool //True - Edgenode False - Cloudnode
+	DefaultToolVer string
+}
+
+//InstallTools sets the OS interface, checks if K8S installation is required or not.
```

Comment:
is k8sInstallation required on edge node ? Even if is required, it should be handled by cloud command I guess ?